.github/workflows/benchmarks.yml

```diff
@@ -10,7 +10,7 @@ permissions:
 
 jobs:
   benchmark:
-    if: ${{ github.event.label.name == 'run-benchmark' && github.event_name == 'pull_request' || github.event_name == 'workflow_dispatch' }}
+    if: contains(github.event.label.name, 'benchmark') || github.event_name == 'workflow_dispatch'
```

Comment:
For those wondering like I did, [`contains` is a GitHub specific expression](https://docs.github.com/en/actions/learn-github-actions/expressions#contains).

Reply:
yeah, I had to do a search to see if there was a comparison function and came across that page :)